src/StreamJsonRpc/JsonRpc.cs

```
@@ -773,13 +773,13 @@ private static JsonRpcMessage CreateError(JToken id, Exception exception)
                 foreach (var targetMap in this.targetRequestMethodToClrMethodMap)
                 {
                     targetMethod = new TargetMethod(request, targetMap.Item1, jsonSerializer, targetMap.Item2);
-                    if (targetMethod.IsFound)
+                    if (targetMethod != null && targetMethod.IsFound)
                     {
                         break;
                     }
                 }
 
-                if (!targetMethod.IsFound)
+                if (targetMethod == null || !targetMethod.IsFound)
```

Thanks for this change. Can we add a test to prove we never throw NRE when `targetMethod==null`, since evidently we are missing one?